pkg/kademlia/lookup_test.go

```diff
@@ -3,39 +3,31 @@
 
 package kademlia_test
 
-import (
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-
-	"storj.io/storj/internal/testcontext"
-	"storj.io/storj/internal/testplanet"
-)
-
-func TestLookupNodes(t *testing.T) {
-	ctx := testcontext.New(t)
-	defer ctx.Cleanup()
-
-	planet, err := testplanet.New(t, 1, 8, 0)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer ctx.Check(planet.Shutdown)
-
-	planet.Start(ctx)
-
-	k := planet.Satellites[0].Kademlia.Service
-	k.WaitForBootstrap() // redundant, but leaving here to be clear
-
-	seen, err := k.DumpNodes(ctx)
-	if err != nil {
-		t.Fatal(err)
-	}
-	assert.NotEqual(t, len(seen), 0)
-	assert.NotNil(t, seen)
-
-	target := seen[0]
-	found, err := k.FindNode(ctx, target.Id)
-	assert.NoError(t, err)
-	assert.Equal(t, target.Id, found.Id)
-}
+//
+//func TestLookupNodes(t *testing.T) {
+//	ctx := testcontext.New(t)
+//	defer ctx.Cleanup()
+//
+//	planet, err := testplanet.New(t, 1, 8, 0)
+//	if err != nil {
+//		t.Fatal(err)
+//	}
+//	defer ctx.Check(planet.Shutdown)
+//
+//	planet.Start(ctx)
+//
+//	k := planet.Satellites[0].Kademlia.Service
+//	k.WaitForBootstrap() // redundant, but leaving here to be clear
+//
+//	seen, err := k.DumpNodes(ctx)
+//	if err != nil {
+//		t.Fatal(err)
+//	}
+//	assert.NotEqual(t, len(seen), 0)
+//	assert.NotNil(t, seen)
+//
+//	target := seen[0]
+//	found, err := k.FindNode(ctx, target.Id)
+//	assert.NoError(t, err)
+//	assert.Equal(t, target.Id, found.Id)
```

Comment:
whats up with this commented out test? should we delete this file?

Reply:
yeah I will delete it. I will be removing all the kad files in a separate PR too